samples/asgardeo-react-app/.env

```diff
@@ -1 +1,4 @@
 PORT=3000
+HOST="localhost"
+DISABLE_DEV_SERVER_HOST_CHECK=false
+DISABLE_DEV_SERVER_HTTPS=false
```

Comment:
Let's just have `HTTPS`.
CRA also uses the term `HTTPS` and it'll be a familiar experience for users.

https://create-react-app.dev/docs/using-https-in-development/

Reply:
Resolved with https://github.com/asgardeo/asgardeo-auth-react-sdk/pull/164/commits/d097f8feddf31f7330e2f3a04694fdc17d15343f